### ISSUE TITLE
N'th Fibonacci number in logarithmic time.

### ISSUE THREAD

**Issue by akshitagupta15june** [akshitagit/CPP#15]
Add in DailyCode repo of CPP

**Comment by trivediavinash** [comment]
@akshitagupta15june Hey can you assign this to me for hactoberfest?

**Comment by akshitagupta15june** [comment]
Fork the repo and add the code , then raise a pull request. I will merge it.

**Comment by trivediavinash** [comment]
@akshitagupta15june  yeah sure can i make pull request after hacktoberfest begin ,pls assign me if you can

**Comment by iaish-max** [comment]
I am a beginner in open source contribution , can i work on this.

**Comment by kratikaatgithub** [comment]
Hi can you please assign this to me as a part of hactoberfest.
Thanks

**Comment by akshitagupta15june** [comment]
fork repo add your code and create a pull request
